src/components/WalletDropdown/DefaultMenu.tsx

```diff
@@ -0,0 +1,258 @@
+  color: ${({ theme }) => theme.textTertiary};
+  margin-top: 8px;
+`
+
+const AuthenticatedHeader = () => {
+  const { account, chainId, connector } = useWeb3React()
+  const [isCopied, setCopied] = useCopyClipboard()
+  const copy = useCallback(() => {
+    setCopied(account || '')
+  }, [account, setCopied])
+  const dispatch = useAppDispatch()
+  const balanceString = useCurrencyBalanceString(account ?? '')
+  const {
+    nativeCurrency: { symbol: nativeCurrencySymbol },
+    explorer,
+  } = getChainInfoOrDefault(chainId ? chainId : SupportedChainId.MAINNET)
+
+  const { address: parsedAddress } = useENS(account)
+  const unclaimedAmount: CurrencyAmount<Token> | undefined = useUserUnclaimedAmount(parsedAddress)
+  const connectionType = getConnection(connector).type
+  const nativeCurrency = useNativeCurrency()
+  const nativeCurrencyPrice = useStablecoinPrice(nativeCurrency ?? undefined) || 0
+
+  const amountUSD = useMemo(() => {
+    const price = parseFloat(nativeCurrencyPrice.toFixed(5))
+    const balance = parseFloat(balanceString || '0')
+    return price * balance
+  }, [balanceString, nativeCurrencyPrice])
+
+  return (
+    <>
+      <div style={{ marginBottom: 12 }}>
+        <div style={{ display: 'inline-block', marginTop: 4 }}>
+          <div style={{ display: 'flex' }}>
+            <StatusIcon connectionType={connectionType} size={24} />
+            <Text fontSize={16} fontWeight={600} marginTop="2.5px">
+              {account && shortenAddress(account, 2, 4)}
+            </Text>
+          </div>
+        </div>
+        <IconContainer>
+          <IconButton onClick={copy} Icon={Copy} text={isCopied ? <Trans>Copied!</Trans> : <Trans>Copy</Trans>} />
+          <IconButton
+            onClick={() => window.open(`${explorer}address/${account}`, '_blank')}
+            Icon={ExternalLink}
+            text={<Trans>Explore</Trans>}
+          />
+          <IconButton
+            onClick={() => {
+              if (connector && connector.deactivate) {
+                connector.deactivate()
+              }
+              connector.resetState()
+              dispatch(updateSelectedWallet({ wallet: undefined }))
+            }}
+            Icon={Power}
+            text={<Trans>Disconnect</Trans>}
+          />
+        </IconContainer>
+      </div>
+      <Column>
+        <div style={{ padding: '16px 0' }}>
+          <Text fontSize={36} fontWeight={400}>
+            {balanceString} {nativeCurrencySymbol}
+          </Text>
+          <USDText>${amountUSD.toFixed(2)} USD</USDText>
+        </div>
+        {unclaimedAmount !== undefined && unclaimedAmount?.toFixed(0, { groupSeparator: ',' } ?? '-') !== '0' && (
+          <UNIbutton style={{ border: 'none' }}>
+            <Trans>Claim</Trans> {unclaimedAmount?.toFixed(0, { groupSeparator: ',' } ?? '-')} <Trans>reward</Trans>
+          </UNIbutton>
+        )}
+      </Column>
+    </>
+  )
+}
+
+const PendingBadge = styled.span`
+  background-color: #4c82fb3d;
```

Comment:
you'll want to use the theme colors from `useTheme`. That handles dark/light mode for you